modules/ROOT/pages/mergetags.adoc

```diff
@@ -74,7 +74,7 @@ Content containing the specified prefix and suffix, and matching a specified mer
 +
 The `+mergetags_list+` option allows for the specification of a nested menu item within the merge tags toolbar menu button. This option allows any number of nested menus and items for merge tag categorisation.
 
-== Styling Merge tags
+== Styling Merge Tags
 
 It's possible to change the visual appearance of the Merge tags within the editor using a custom CSS file provided through the xref:add-css-options.adoc#content_css[content_css] option. Here is an example on how to style the various elements of the merge tag.
```

Comment:
```suggestion
It's possible to change the visual appearance of the Merge Tags within the editor using a custom CSS file provided through the xref:add-css-options.adoc#content_css[content_css] option. Here is an example of how to style the various elements of the merge tag.
```

Reply:
There's actually quite a bit of inconsistent use of `merge tags`, `Merge tags` and `Merge Tags` in this file as I noted in the call so we probably need to look at the whole file.

Reply:
> There's actually quite a bit of inconsistent use of `merge tags`, `Merge tags` and `Merge Tags` in this file as I noted in the call so we probably need to look at the whole file.

Already got a ticket for just that [DOC-1821](https://ephocks.atlassian.net/browse/DOC-1821).